usd_qtpy/render_util/playblast.py

```diff
@@ -0,0 +1,166 @@
+# Playblast framework
+# Inspired by: Prism, usdrecord
+
+# NOTES:
+# pxr.UsdViewq.ExportFreeCameraToStage will export the camera from the view (a FreeCamera/ pxr.Gf.Camera, purely OpenGL)
+
+from pxr import Usd, UsdGeom
+from pxr import UsdAppUtils
+from pxr import Tf, Sdf
+
+from pxr.Usdviewq.stageView import StageView
+from viewer import CustomStageView
+
+from qtpy import QtCore
+
+from typing import Union
+import logging
```

Comment:
Can we reorder the imports to be:

- first standard built-ins
- dependencies
- local imports

Also ensure local relative imports are forced to be relative imports via e.g. `.viewer import CustomStageView`.

```suggestion
import logging
from typing import Union

from qtpy import QtCore
from pxr import Usd, UsdGeom
from pxr import UsdAppUtils
from pxr import Tf, Sdf
from pxr.Usdviewq.stageView import StageView

from .viewer import CustomStageView
```

Reply:
I reordered everything, it makes sense.

`from .viewer import CustomStageView` will actually not resolve as I have defined render_util as a complete module.
I plan to add turntable functionality in a seperate file later, so I wanted to keep those seperate.

```py
from ..viewer import CustomStageView
```
Does resolve, so I will leave it as such.